docs/modules/ROOT/pages/cops_style.adoc

```diff
@@ -961,6 +967,12 @@ class A
     @@test # you can access class variable without offense
   end
 end
+
+class A
+  def test(name)
+    class_variable_get(":@@#{name}") # you can access without offense
```

Comment:
I think this example will occur a `NoMethodError`.

```ruby
class Foo
  def do_something
    class_variable_get('@@var') # you can access without offense
  end
end

Foo.new.do_something
```

```console
% ruby example.rb
Traceback (most recent call last):
        1: from example.rb:7:in `<main>'
example.rb:3:in `do_something': undefined method `class_variable_get' for #<Foo:0x00007feb088407d8> (NoMethodError)
```

Reply:
Sorry, I should have actually run my own examples before committing. Updated to use class methods and for each bad/good example to be complete.